.github/workflows/cross-version-tests.yml

```diff
@@ -117,6 +117,7 @@ jobs:
           repository: ${{ github.event.inputs.repository }}
           ref: ${{ github.event.inputs.ref }}
       - uses: ./.github/actions/free-disk-space
+        if: matrix.package == 'transformers'
```

Comment:
sentence-transformers flavor will need this too. It pulls a very large model to do much of its testing

Reply:
Got it, I will add it.